### PR TITLE
Add data import documents

### PR DESCRIPTION
**Pull request by VadymSachenko**
## PR Description
This PR adds new data import documents.
Jira task: https://spryker.atlassian.net/browse/CC-25233
TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: _drafts/data-import/202309.0/file-details-shipment-method-store.csv.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+description: This document describes the shipment_method_store.csv file to configure the Shipment information in your Spryker Demo Shop.
+template: data-import-template
+last_updated: May 23, 2023
+---
+
+This document describes the `shipment_method_store.csv` file to configure the [shipment](/docs/pbc/all/carrier-management/base-shop/shipment-feature-overview.html) type information in your Spryker Demo Shop.
+
+## Import file dependencies
+
+[TODO: verify dependencies and add missing ones; if no dependencies, remove the section]
```
Comment (VadymSachenko) [inline _drafts/data-import/202309.0/file-details-shipment-method-store.csv.md:12]: Resolve TODO

**Review by AsonUnique — APPROVED** [review]

File: docs/pbc/all/carrier-management/202304.0/base-shop/import-and-export-data/file-details-shipment-method-shipment-type.csv.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+description: This document describes the shipment_method_shipment_type.csv file to configure the Shipment information in your Spryker Demo Shop.
+template: data-import-template
+last_updated: May 23, 2023
+---
+
+This document describes the `shipment_method_shipment_type.csv` file to configure the [shipment](/docs/pbc/all/carrier-management/base-shop/shipment-feature-overview.html) type information in your Spryker Demo Shop.
+
+## Import file dependencies
+
+[`shipment.csv`](/docs/pbc/all/carrier-management/{{page.version}}/base-shop/import-and-export-data/file-details-shipment.csv.html)
```
Comment (VadymSachenko) [inline docs/pbc/all/carrier-management/202304.0/base-shop/import-and-export-data/file-details-shipment-method-shipment-type.csv.md:12]:
```suggestion
* [`shipment.csv`](/docs/pbc/all/carrier-management/{{page.version}}/base-shop/import-and-export-data/file-details-shipment.csv.html)
```

File: docs/pbc/all/carrier-management/202304.0/base-shop/import-and-export-data/file-details-shipment-method-shipment-type.csv.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+[`shipment-type.csv`](/docs/pbc/all/carrier-management/{{page.version}}/base-shop/import-and-export-data/file-details-shipment-type.csv.html)
```
Comment (VadymSachenko) [inline docs/pbc/all/carrier-management/202304.0/base-shop/import-and-export-data/file-details-shipment-method-shipment-type.csv.md:13]:
```suggestion
* [`shipment-type.csv`](/docs/pbc/all/carrier-management/{{page.version}}/base-shop/import-and-export-data/file-details-shipment-type.csv.html)
```

**Review by andriispryker — DISMISSED** [review]
Approved.

File: docs/pbc/all/carrier-management/202304.0/base-shop/import-and-export-data/file-details-shipment-method-shipment-type.csv.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+---
+title: File details - shipment_method_shipment_type.csv
+description: This document describes the shipment_method_shipment_type.csv file to configure the Shipment information in your Spryker Demo Shop.
```
Comment (lenadoc) [inline docs/pbc/all/carrier-management/202304.0/base-shop/import-and-export-data/file-details-shipment-method-shipment-type.csv.md:3]:
```suggestion
description: This document describes the shipment_method_shipment_type.csv file to configure the shipment information in your Spryker Demo Shop.
```

File: docs/pbc/all/carrier-management/202304.0/base-shop/import-and-export-data/file-details-shipment-type-store.csv.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+---
+title: File details - shipment_type_store.csv
+description: This document describes the shipment_type_store.csv file to configure the Shipment information in your Spryker Demo Shop.
```
Comment (lenadoc) [inline docs/pbc/all/carrier-management/202304.0/base-shop/import-and-export-data/file-details-shipment-type-store.csv.md:3]:
```suggestion
description: This document describes the shipment_type_store.csv file to configure the shipment information in your Spryker Demo Shop.
```

File: docs/pbc/all/carrier-management/202304.0/base-shop/import-and-export-data/file-details-shipment-type.csv.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
+---
+title: File details - shipment_type.csv
+description: This document describes the shipment_type.csv file to configure the Shipment information in your Spryker Demo Shop.
```
Comment (lenadoc) [inline docs/pbc/all/carrier-management/202304.0/base-shop/import-and-export-data/file-details-shipment-type.csv.md:3]:
```suggestion
description: This document describes the shipment_type.csv file to configure the shipment information in your Spryker Demo Shop.
```

**Review by lenadoc — CHANGES_REQUESTED** [review]

**Review by lenadoc — APPROVED** [review]

**Review by lenadoc — APPROVED** [review]

**Pull request merged**